lib/singlepass-backend/src/emitter_x64.rs

```diff
@@ -168,6 +170,16 @@ pub trait Emitter {
     fn emit_call_location(&mut self, loc: Location);
 
     fn emit_bkpt(&mut self);
+
+    fn emit_homomorphic_host_redirection(&mut self, target: GPR);
```

Comment:
I don't understand what this function is for from the name. Could you add a comment?